Makefile

```diff
@@ -12,4 +17,18 @@ clean:
 
 .PHONY: run
 run:
-	go run cmd/server/main.go
\ No newline at end of file
+	go run cmd/server/main.go
+
+.PHONY: migration_up
+migration_up:
+	migrate -source file://db/migrations -database "postgres://$(DATABASE_USER):$(DATABASE_PASSWORD)@localhost:5432/$(DATABASE_NAME)?sslmode=disable" up $(if $(N),$(N),)
+
+.PHONY: migration_down
+migration_down:
+	migrate -source file://db/migrations -database "postgres://$(DATABASE_USER):$(DATABASE_PASSWORD)@localhost:5432/$(DATABASE_NAME)?sslmode=disable" down $(if $(N),$(N),)
+
+.PHONY: create_migration
+create_migration:
+	migrate create -ext sql -dir /db/migrations  $(NAME)
```

Comment:
Here the command won't work properly because the directory is wrong. Either put `db/migration` or `./db/migrations` here, because `/db/migrations` will look for a `db` directory in the root of the file system, not the current working directory.